db-compare-server/dev-build.sh

```diff
@@ -0,0 +1,10 @@
+  $(killall db-compare-server)
+clear
+  go build && echo " \e[1m\e[36mapp rebuilt successfully\e[0m" || echo -e "\n\n\e[1m[91mrebuild failed\n\n"
+  ./db-compare-server & 
+inotifywait ./ -q -m -e close_write | while read events; do 
```

Comment:
`inotifywait` is not installed by default. Need instructions for installation.

Reply:
`fswait` might be a better cross-platform alternative https://stackoverflow.com/questions/1515730/is-there-a-command-like-watch-or-inotifywait-on-the-mac

Reply:
updated to use fswatch....an interesting little piece of work.